bundles/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/internal/handler/OpenUVBridgeHandler.java

```diff
@@ -17,11 +17,14 @@
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.util.Collection;
+import java.util.Optional;
 import java.util.Properties;
 import java.util.Set;
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
 
+import javax.ws.rs.HttpMethod;
```

Comment:
I would suggest to rather use org.eclipse.jetty.http.HttpMethod

Reply:
Correct.

Reply:
Unchanged

Reply:
It is removed no ?

Reply:
Yes, I meant "Change undone" rather than "unchanged".

Reply:
Yes, I kept it as is at the end.